.github/workflows/quality-checks.yml

```diff
@@ -35,3 +35,45 @@ jobs:
     - name: Test
       run: |
         scripts/test
+
+  validation_build_ubuntu_python:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        scripts/install_deps
+
+    - name: Build
+      run: |
+        scripts/build
+        
+  validation_docker_ubuntu:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Build
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
```

Comment:
We are building releases with Dockerfile.release. Testing now with Dockerfile. Should this run `--target=test`?

What should happen here?

Reply:
Good point. Adding release docker validation